acceptance/tests/sameness/sameness_test.go

```diff
@@ -0,0 +1,443 @@
+				createNamespaces(t, cfg.NoCleanupOnFailure, v.context)
+			}
+
+			// Create the Default Cluster.
+			commonHelmValues := map[string]string{
+				"global.peering.enabled": "true",
+
+				"global.tls.enabled":   "true",
+				"global.tls.httpsOnly": strconv.FormatBool(c.ACLsEnabled),
+
+				"global.enableConsulNamespaces": "true",
+
+				"global.adminPartitions.enabled": "true",
+
+				"global.logLevel": "debug",
+
+				"global.acls.manageSystemACLs": strconv.FormatBool(c.ACLsEnabled),
+
+				"connectInject.enabled":                       "true",
+				"connectInject.consulNamespaces.mirroringK8S": "true",
+
+				"meshGateway.enabled":  "true",
+				"meshGateway.replicas": "1",
+
+				"dns.enabled": "true",
+			}
+
+			defaultPartitionHelmValues := map[string]string{
+				"global.datacenter": primaryServerDatacenter,
+			}
+
+			// On Kind, there are no load balancers but since all clusters
+			// share the same node network (docker bridge), we can use
+			// a NodePort service so that we can access node(s) in a different Kind cluster.
+			if cfg.UseKind {
+				defaultPartitionHelmValues["meshGateway.service.type"] = "NodePort"
+				defaultPartitionHelmValues["meshGateway.service.nodePort"] = "30200"
+				defaultPartitionHelmValues["server.exposeService.type"] = "NodePort"
+				defaultPartitionHelmValues["server.exposeService.nodePort.https"] = "30000"
+				defaultPartitionHelmValues["server.exposeService.nodePort.grpc"] = "30100"
+			}
+			helpers.MergeMaps(defaultPartitionHelmValues, commonHelmValues)
+
+			releaseName := helpers.RandomName()
+			members[keyPrimaryServer].helmCluster = consul.NewHelmCluster(t, defaultPartitionHelmValues, members[keyPrimaryServer].context, cfg, releaseName)
+			members[keyPrimaryServer].helmCluster.Create(t)
+
+			// Get the TLS CA certificate and key secret from the server cluster and apply it to the client cluster.
+			caCertSecretName := fmt.Sprintf("%s-consul-ca-cert", releaseName)
+
+			logger.Logf(t, "retrieving ca cert secret %s from the server cluster and applying to the client cluster", caCertSecretName)
+			k8s.CopySecret(t, members[keyPrimaryServer].context, members[keyPartition].context, caCertSecretName)
+
+			// Create Secondary Partition Cluster which will apply the primary datacenter.
+			partitionToken := fmt.Sprintf("%s-consul-partitions-acl-token", releaseName)
+			if c.ACLsEnabled {
+				logger.Logf(t, "retrieving partition token secret %s from the server cluster and applying to the client cluster", partitionToken)
+				k8s.CopySecret(t, members[keyPrimaryServer].context, members[keyPartition].context, partitionToken)
+			}
+
+			partitionServiceName := fmt.Sprintf("%s-consul-expose-servers", releaseName)
+			partitionSvcAddress := k8s.ServiceHost(t, cfg, members[keyPrimaryServer].context, partitionServiceName)
+
+			k8sAuthMethodHost := k8s.KubernetesAPIServerHost(t, cfg, members[keyPartition].context)
+
+			secondaryPartitionHelmValues := map[string]string{
+				"global.enabled": "false",
```

Comment:
```suggestion
				"global.enabled":     "false",
				"global.datacenter": primaryServerDatacenter,
```

Reply:
something like this

Reply:
weird, I lint on save

Reply:
oh it's because there is an empty newline, gofmt won't space the values if there's an empty newline